pipelines/qc/Snakefile

```diff
@@ -35,38 +72,27 @@ rule estimate_contamination:
     plot=True,
     log=False
   script: 
-    os.path.join(config["scripts_loc"], "estimate_contamination.py")
+    os.path.join(script_dir, "estimate_contamination.py")
 
 
 rule coverage_summary:
   input:
-    input=lambda y: samples.path.loc[y.sample],
+    input=lambda y: path_df.path.loc[y.sample],
   output:
-    csv="output/{sample}.callstats.csv",
-    npy="output/{sample}.covhist.npz"
+    csv="{sample}.callstats.csv",
+    npy="{sample}.covhist.npz"
   params:
-    stem="output/{sample}",
+    stem="{sample}",
     seqid=config["seqid_stats"]
   script: 
-    os.path.join(config["scripts_loc"], "calculate_alignment_summary_stats.py")
+    os.path.join(script_dir, "calculate_alignment_summary_stats.py")
 
 
-rule combine_individual_outputs_apply_qc_filters:
-  input:
-    manifest=config["manifest"],
-    csv=expand("output/{sample}.callstats.csv", sample=samples.index.tolist()),
-    npy=expand("output/{sample}.covhist.npz", sample=samples.index.tolist()),
-    contam=expand("output/{sample}.contamination.csv", sample=samples.index.tolist())
-  params:
-    min_fgc=config["min_fgc"],
-    min_med_cov=config["min_med_cov"],
-    min_female_xratio=config["min_female_xratio"],
-    max_male_xratio=config["max_male_xratio"],
-    max_pc_contam=config["max_pc_contam"],
-    path=lambda y: "output/{sample}"
-  output:
-    csv="output/qc_summary.csv"
-  script:
-    os.path.join(config["scripts_loc"], "combine_qc_cohort_stats.py")
-    
+# required for qc_report.
+rule make_manifest:
```

Comment:
Alternatively, we could pass the list of samples as an argument, thus avoiding creation of extra file. Are there downsides of this approach?

Reply:
It's slightly awkward to pass lists within snakemake I think.

Reply:
I see logic in that now - thus we keep file listing samples in the output folder. This is reasonable for me.